src/cmd/services/m3coordinator/downsample/flush_handler.go

```diff
@@ -185,6 +185,12 @@ func (w *downsamplerFlushHandlerWriter) Write(
 			},
 		})
 		if err != nil {
+			logger.Error("downsampler flush error constructing write", zap.Error(err))
```

Comment:
nit: `writeQuery` here so we can distinguish where it fails in logs?

Reply:
👍